osu.Game/Graphics/UserInterface/NormalSliderBar.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osuTK;
+using osuTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Events;
+using osu.Game.Overlays;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public partial class NormalSliderBar<T> : OsuSliderBar<T>
```

Comment:
opted for normal instead of "basic" because it caused conflicts with a framework side class